ui/vic-ui-h5c/vic/src/vic-webapp/src/app/create-vch-wizard/security/security.component.ts

```diff
@@ -72,7 +74,16 @@ export class SecurityComponent implements OnInit {
 
     removeTlsCaEntry(index: number) {
```

Comment:
I'm trying to think of a way we can consolidate these functions. We have a lot of repetitive code. Maybe we could pass in the control name? Let me know if it's feasible or not.